Update Safe data

Fbartoli opened the pull request:
Add Ackee corporate entity definition and update software wallets with 'safe' support attribute

- Introduced a new corporate entity for Ackee with relevant details.
- Added 'safe' support attribute to multiple software wallets including Ambire, Coinbase, Daimo, Elytros, Family, Frame, Gem, Metamask, Nufi, Phantom, Rainbow, and Zerion.
- Enhanced account portability evaluation to include Safe multisig accounts.
- Updated account support schema to recognize Safe as a new account type.

Fbartoli committed:
Add Ackee corporate entity definition and update software wallets with 'safe' support attribute

- Introduced a new corporate entity for Ackee with relevant details.
- Added 'safe' support attribute to multiple software wallets including Ambire, Coinbase, Daimo, Elytros, Family, Frame, Gem, Metamask, Nufi, Phantom, Rainbow, and Zerion.
- Enhanced account portability evaluation to include Safe multisig accounts.
- Updated account support schema to recognize Safe as a new account type.

polymutex commented on src/schema/attributes/self-sovereignty/account-portability.ts:
```diff
@@ -409,6 +410,55 @@ function evaluateMultifactor(
+		}
+	}
+
+	// Add other conditions similar to multifactor
```
I assume this is to be implemented in future PRs?

polymutex commented on data/software-wallets/safe.ts:
```diff
@@ -68,22 +95,54 @@ export const safe: SoftwareWallet = {
+			walletCall: supported({
+				ref: {
+					explanation: 'Safe supports EIP-5792 for transaction batching.',
+					url: 'https://github.com/safe-global/safe-modules/tree/main/modules/batching',
```
This appears to be a 404

polymutex reviewed (approved):
Thanks, great to see the Safe account type shaping up :)

Fbartoli committed:
Update software wallets and schema to enhance Safe account support

- Added 'Ackee' and 'RPGF' to the cspell dictionary.
- Updated the rabby and safe wallet definitions to include the TransactionGenerationCapability and improved default configurations.
- Enhanced account support schema to recognize Safe as a new account type and updated transaction batching attributes accordingly.
- Improved account portability evaluation for Safe multisig accounts.

polymutex commented:
Thanks for contributing! This is great :)
